pbm/pbm.go

```diff
@@ -860,3 +860,22 @@ func FileCompression(ext string) CompressionType {
 		return CompressionTypeS2
 	}
 }
+
+// CopyColl copy documents matching the given filter and return number of copied documents
```

Comment:
I thought 'An $out or $merge stages at the end of an aggregation pipeline command can do this', but only from v4.4 can it be output to a collection in another db namespace. So never mind, it's not suitable.